package.json

```diff
@@ -28,6 +29,7 @@
     "@types/node": "^6.0.38",
     "@types/source-map": "^0.1.27",
     "@types/source-map-support": "^0.2.27",
+    "base-64": "^0.1.0",
```

Comment:
Is this library in g3?  We'll have to import it if so.
It might be possible to just use builtin node APIs for base64:
https://nodejs.org/docs/latest/api/buffer.html#buffer_buffers_and_character_encodings